### DISCUSSION TITLE
add unique dialog if you land at geminus at  july 4th 3014 instead of just the standed dialog

### DISCUSSION THREAD

**Discussion by eatsomedust4** [endless-sky/endless-sky#10400] [Ideas]
it would make sence tho

**Comment by mOctave** [comment]
I'm going to try to write something up for this.

**Reply by mOctave** [reply]
https://github.com/endless-sky/endless-sky/pull/10407

**Comment by TheGiraffe3** [comment]
Should this be closed, as #10407 was merged?
